cmd/check/root.go

```diff
@@ -42,13 +44,35 @@ func newCheckCmd() *cobra.Command {
 				return err
 			}
 
+			subject := args[0]
+			relation := args[1]
+			namespace := args[2]
+			object := args[3]
+
+			var s *rts.Subject
+
+			// distinguish between subjectSet and subjectID
+			if strings.Contains(subject, ":") {
+				su := &ketoapi.SubjectSet{}
+				su, err := su.FromString(subject)
+				if err != nil {
+					return err
+				}
+
+				s = rts.NewSubjectSet(su.Namespace, su.Object, su.Relation)
+			} else {
+				s = rts.NewSubjectID(subject)
+			}
```

Comment:
I'd prefer to make this a small helper function that has its own unit test. That way we can specifically test for the subject parsing of the CLI with more edge cases. The e2e test makes sense as it is right now as well just to ensure that subject sets are parsed and supported by all clients.